lib/horoshop.rb

```diff
@@ -1,7 +1,11 @@
 # frozen_string_literal: true
 
+require 'faraday'
```

Comment:
This row should be extracted from this class. In all next classes we need faraday gem too.